src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/SetAzureSqlFailoverGroup.cs

```diff
@@ -50,9 +51,11 @@ public class SetAzureSqlFailoverGroup : AzureSqlFailoverGroupCmdletBase
         /// <summary>
         /// Gets or sets the grace period with data loss for the Sql Azure Failover Group.
         /// </summary>
+        [Alias("GracePeriodWithDataLossHours")]
```

Comment:
@ziwa-msft please remove this. This will still raise the obsolete warning no matter what parameter name user uses.
Introduce the new name when you make the breaking change. Right now simply add the warning.